# Remove use of -Werror because Werror is not used (discussed in http://seqanswers.com/forums/showthread.php?t=20113). 
# Using -Werror gives warnings which result in a failure of the installation.
# Author: Fokke Dijkstra - Rijksuniversiteit Groningen (RUG)
--- lastz-distrib-1.02.00/src/Makefile	2010-01-13 18:00:09.000000000 +0100
+++ lastz-distrib-1.02.00/src/Makefile	2015-02-09 17:22:41.030256461 +0100
@@ -28,7 +28,7 @@
 #
 #---------
 
-definedForAll = -Wall -Wextra -Werror -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE
+definedForAll = -Wall -Wextra -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE
 
 
 VERSION_FLAGS= \

